### DISCUSSION TITLE
江西省抚州市东乡区红星雅苑 10年未能办理房产证

### DISCUSSION THREAD

**Discussion by viniser** [WeNeedHome/SummaryOfLoanSuspension#565] [General]
江西紫海方舟实业有限公司 法定代表人：胡耀平
江西省抚州市东乡区红星雅苑 10年未能办理房产证
该公司私自挪用监管资金导致烂账 在
江西省抚州市东乡区
江西省抚州市宜黄县。大量问题商品房
上百条司法诉讼，替罪羊法人进入失信人黑名单开始摆烂。

**Comment by Swiftfrog** [comment]
那些心存幻想的人呢？看一下，10年，你有几个10年？
